packages/contracts/contracts/LenderCommitmentForwarder.sol

```diff
@@ -232,6 +252,20 @@ contract LenderCommitmentForwarder is TellerV2MarketForwarder {
         );
     }
 
+      function updateAllowlistManager(
+        uint256 _commitmentId,
+        address _allowlistManager
+    ) public commitmentLender(_commitmentId) {
+       
+        commitmentAllowListManagers[_commitmentId] = _allowlistManager;
+
+        emit UpdatedAllowlistManager(_commitmentId,_allowlistManager);
+        //emit UpdatedAllowList(_commitmentId);
+    }
+
+
+//should be able to set this in one tx 
```

Comment:
should be able to migrate legacy commitments in one TX to prevent a tx in between